apps/mdn/mdn-aws/k8s/regions/germany/prod.mm.sh

```diff
@@ -51,7 +51,7 @@ export WEB_CPU_LIMIT=4
 export WEB_CPU_REQUEST=500m
 export WEB_MEMORY_LIMIT=6Gi
 export WEB_MEMORY_REQUEST=4Gi
-export WEB_ALLOWED_HOSTS="developer.mozilla.org,cdn.mdn.mozilla.net,mdn.mozillademos.org,demos.mdn.mozit.cloud,demos-origin.mdn.mozit.cloud,developer-prod.mdn.mozit.cloud,prod.mdn.mozit.cloud,standby.mdn.mozit.cloud,developer.mozilla.com,mdn.mozilla.org,developer-new.mozilla.org,developers.mozilla.org"
```

Comment:
@limed - Are there additional IT steps to get rid of these domains? There's [bug 1423995](https://bugzilla.mozilla.org/show_bug.cgi?id=1423995) to add the CNAME, should we open a new bug to drop the CNAMEs or just CC on [bug 1503261](https://bugzilla.mozilla.org/show_bug.cgi?id=1503261)?